homeassistant/components/sensor/miflora.py

```diff
@@ -15,7 +21,22 @@
     CONF_FORCE_UPDATE, CONF_MONITORED_CONDITIONS, CONF_NAME, CONF_MAC
 )
 
+
+class _BACKENDS(Enum):
+    """Symbolic constants for the different bluetooth backends."""
+
+    GATTTOOL = 0
+    BLUEPY = 1
+
+
 REQUIREMENTS = ['miflora==0.2.0']
+# normally use gatttool as this is what people are used to
+_BACKEND = _BACKENDS.GATTTOOL
+# on linux use bluepy
+if sys.platform == "linux" or sys.platform == "linux2":
+    REQUIREMENTS.append('bluepy==1.1.4')
+    _BACKEND = _BACKENDS.BLUEPY
+
```

Comment:
Can you remove this stuff and make bluepy as default backend. Maybe you can make a config options that user can overwrite this with config without this Enum and static stuff.

Reply:
Are you really sure, that we want to make it the default backend? This might not work on all platforms...

Reply:
@pvizeli and how do you want to make the REQUIREMENTS configurable? When this statement is parsed during the startup of home assistant, I do not yet have access to the config file...

Reply:
I can't. REQUIREMENTS are all time static. You can check inside `setup` if bluepy are importable otherwise you use gattool and don't but it into REQUIREMENTS. You can explain on documentation that he will use bluepy if they is available.

Reply:
@pvizeli I agree with you. When I think about the normal hassio user (=not a developer): How would he install bluepy? Doing it via pip is not an option for the normal user, I would say.

I see these options:
1. We create a dummy component, that does nothing except depending on bluepy. So if you use this component in your confing, you will have bluepy in your installation and it can be used by miflora.
2. We create a second component e.g. ```miflora_bluepy``` that uses bluepy instead of gatttool. It might be interesting here to avoid duplicating the code.
3. We use the solution I proposed in this PR.

Do you have any other ideas?

Reply:
HassIO have preinstall bluepy and gatttool (Yes, on image 0.60 it have a bug but that is fix with 0.61)

Reply:
OK, then it should work. I updated the code that decides on the Backend to be used.